stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -328,8 +328,7 @@ BitMask BuiltinTypeInfo::getSpareBits(TypeConverter &TC, bool &hasAddrOnly) cons
     mask.keepOnlyMostSignificantBits(getSize() * 8 - intSize);
     return mask;
   } else if (
-    Name == "ypXp" || // Any.Type
-    Name == "yyXf" // 'yyXf' =  @thin () -> Void function
```

Comment:
For my own understanding, why can you remove this case from here now?

Reply:
Because real 'yyXf' pointers don't expose spare bits.   RemoteMirror handles some existential types by pretending they're built out of these function pointers, but that's not how the compiler models it.   This goes along with the changes below to `RecordTypeInfo::getSpareBits` that directly expose spare bits for these existential types without relying on this logic.